src/persist_book.rb

```diff
@@ -0,0 +1,26 @@
+require './src/book'
+require './src/modules/list'
+require 'json'
+
+class PersistBook
+  def initialize(book)
+    @books = book
+  end
+  include List
+
+  def read_book_data
+    book_data = File.read('./src/books.json')
```

Comment:
- [ ] Kindly handle the case when one or more of the files are missing (the app should work and show no errors). By adding this line: ` return [] unless File.exists?("./src/books.json")` for all files to check file exists or not.

Reply:
make the changes accordingly, Thank you for the valuable comment